### DISCUSSION TITLE
固液界面单词学习1 - Dr. Jun Xia

### DISCUSSION THREAD

**Discussion by giscus** [Jungeson/jungeson.github.io#2] [Announcements]
# 固液界面单词学习1 - Dr. Jun Xia

hydrodynamic 流体动力学entrance length 管道入口段（起始段）velocity profile 流速剖面（泊肃叶流的流速为抛物线）viscous forces 粘性力 粘滞力propagate 扩散 传播interior wall 管的内壁no-slip condition 无滑移条件motion 运动adjacent 相邻的conservation of mass 质量守恒compensate 补偿 弥补cross-section 横截面boundary layer 边界层

http://staff.ustc.edu.cn/~xiajun/posts/2022/05/09/1/

**Comment by ljh13892912343** [comment]
## 固液界面学习必备
码了码了！(☆▽☆)

**Reply by Jungeson** [reply]
喵
